roles/ngi_reports/tasks/dependencies.yml

```diff
@@ -18,25 +18,25 @@
   tags: ngi_reports
 
 - name: Unpack TeXLive
-  unarchive:
-    src: "{{ texLive_dest }}/src/{{ texLive_file }}"
-    dest: "{{ texLive_dest }}/src/"
-    copy: no
-    creates: "{{ texLive_dest }}/src/install-tl-*/LICENSE.TL"
-    mode: g+rw
+  command: "tar -xzf {{ texLive_dest }}/src/{{ texLive_file }} -C {{ texLive_dest }}/src/ --strip-components=1"
```

Comment:
Note also that if you're using the `command` instead of the `shell` module then I usually specify the whole path to the binary, as it won't generally have the binary in the `$PATH`.